src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -100,10 +100,14 @@ public void Configure(string name, OpenIdConnectOptions options)
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
             var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
-            {
-                _logger.LogWarning("The OpenID Connect module is not correctly configured.");
```

Comment:
Sorry guys, this line shouldn't go, as now you no longer know where the error is coming from.

If you want to log the validation errors as well, they should be concated together here

Here's a quick example from elsewhere
```
                            if (_logger.IsEnabled(LogLevel.Warning))
                            {
                                _logger.LogWarning("The OpenId Connect module is not correctly configure : '{Errors}'", string.Join(", ", result.Errors));
                            }
```

Reply:
@deanmarcussen, hmm I didn't see much changes from the previous code, except he logs the inner error message instead of "The OpenID Connect module is not correctly configured."

Reply:
@hishamco multiple inner error message**S** and without any context of where those error messages are from

Reply:
Hi ， I have commit the  new codes , please help me verify.